build.sbt

```diff
@@ -0,0 +1,28 @@
+
+name := "support-admin-console"
+
+version := "1.0-SNAPSHOT"
+
+scalaVersion := "2.12.8"
+
+resolvers += "Guardian Platform Bintray" at "https://dl.bintray.com/guardian/platforms"
```

Comment:
oh interesting, are we using Bintray by default these days? or is this just for certain dependencies?

Reply:
It's for `simple-configuration`, but not sure why bintray was chosen